api-gateway/src/main/java/com/petclinic/bffapigateway/domainclientlayer/VisitsServiceClient.java

```diff
@@ -2,10 +2,11 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.petclinic.bffapigateway.dtos.Visits.*;
```

Comment:
Doesn't this import make it more simple since it imports everything in Visits instead of having to import every single part of it?
It might be better to keep it, in my opinion

Reply:
I optimized the imports for the visit-service-new and the related visit files in the api-gateway to only include necessary imports, which is a good practice to prevent unnecessary importation from occurring.

Reply:
I see
Thanks